lib/Widgets/Toast.vala

```diff
@@ -171,4 +171,14 @@ public class Granite.Toast : Gtk.Widget {
         stop_timeout ();
         start_timeout ();
     }
+
+    /**
+     * Withdraws the currently displayed Toast
+     * @since 7.4.0
+     */
+    [Version (since = "7.4.0")]
+    public void withdraw_notification () {
```

Comment:
Why not call it simply `withdraw`?

Reply:
I think the idea was to model the API after Gtk.Application? But yeah I'm not opposed to just calling it "withdraw". I'll update it